runtime/queries/luau/injections.scm

```diff
@@ -0,0 +1 @@
+(comment) @comment
```

Comment:
```suggestion
((comment) @injection.content
 (#set! injection.language "comment"))
```

The `(comment) @comment` style is an older way that Nvim used to do injections. We follow https://tree-sitter.github.io/tree-sitter/3-syntax-highlighting.html?highlight=inject#language-injection instead